src/pages/publication/DeletePublicationModal.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FC, useState } from 'react';
+import Modal from './../../components/Modal';
+import { useTranslation } from 'react-i18next';
+import { DialogActions, DialogContent, Button } from '@material-ui/core';
+import styled from 'styled-components';
+import NormalText from './../../components/NormalText';
+
+const StyledTitleContainer = styled.div`
+  font-style: italic;
+`;
+
+interface DeletePublicationModalProps {
+  id: string;
+  title: string;
+  setDeletePublicationId: (id: string) => void;
+}
+
+const DeletePublicationModal: FC<DeletePublicationModalProps> = ({ id, title, setDeletePublicationId }) => {
+  const [openModal, setOpenModal] = useState(!!id);
+  const { t } = useTranslation();
+
+  const cancelDelete = () => {
+    setDeletePublicationId('');
+    setOpenModal(false);
+  };
+
+  const deletePublication = () => {
+    // delete publication here
+    setDeletePublicationId('');
+    setOpenModal(false);
+  };
```

Comment:
skal `cancelDelete` og `deletePublication` være helt lik?

Reply:
kan vi ikke droppe `setDeletePublicationId` som prop også? trenger bare `openModal` fra state for å bestemme om modalen skal vises eller ikke

Reply:
VI har ikke noen tjeneste for sletting ennå så foreløpig er de det.